src/stdune/option.mli

```diff
@@ -14,6 +14,8 @@ module O : sig
   val ( let+ ) : 'a t -> ('a -> 'b) -> 'b t
 end
 
+val elim : 'a t -> 'c -> f:('a -> 'c) -> 'c
```

Comment:
We call this `value_map` in jane, and the "none" handler is called `default`. I think we should keep to base naming scheme where possible.

Reply:
Yeah, but the convention is base is not an improvement. We can define an appropriate `elim` many data types, while `value_map` is option specific.